docs/integrations/persisting-store-data.md

```diff
@@ -312,9 +334,10 @@ useBoundStore.persist.getOptions().name
 
 ### `setOptions`
 
-> Schema: `(newOptions: PersistOptions) => void`
+> Type: `(newOptions: Partial<PersistOptions>) => void`
```

Comment:
I aligned the type with [persist.ts@77](https://github.com/pmndrs/zustand/blob/ca059788d3f015b1582dc4d82b2bd4dbb6d93de4/src/middleware/persist.ts#L77).